src/main/java/com/infamous/combat_next/mixin/LivingEntityMixin.java

```diff
@@ -85,7 +85,7 @@ private void fixBlockedByShield(LivingEntity defender, CallbackInfo ci){
         if(BugFixConfigs.fixShieldAttackerKnockback.get()){
             ci.cancel();
             // Fix MC-147694 by switching attacker and defender
-            this.knockback(0.5D, this.getX() - defender.getX(), this.getZ() - defender.getZ());
+            this.knockback(0.5D, defender.getX() - this.getX(), defender.getZ() - this.getZ());
```

Comment:
Is this not just the original code then? Makes the config rather pointless

Reply:
no? this is running it on the attacker instead of the target, which is different from normal

Reply:
it would be vanilla if the line was `this.knockback(0.5D, defender.getX() - this.getX(), defender.getZ() - this.getZ());`